src/dir_entry.rs

```diff
@@ -111,3 +112,25 @@ impl Ord for DirEntry {
         self.path().cmp(other.path())
     }
 }
+
+impl Colorable for DirEntry {
+    fn path(&self) -> PathBuf {
+        self.path().to_owned()
+    }
+
+    fn file_name(&self) -> OsString {
+        let name = match &self.inner {
+            DirEntryInner::Normal(e) => e.file_name(),
+            DirEntryInner::BrokenSymlink(_) => todo!(),
```

Comment:
This todo should probably be done before merging. 

I imagine it could probably be something like:


```suggestion
            DirEntryInner::BrokenSymlink(p) => p.file_name().map_or_else(OsString::new, OsStr::to_os_string),
```

well, I guess then you might need to move the `to_owned` up to line 123 as well.

Reply:
oh, I totally missed that

Reply:
Oh whoops.  I wrote this commit a while ago and it was marked WIP.  Couldn't figure out why...

Reply:
This is the right implementation: https://github.com/sharkdp/lscolors/blob/master/src/lib.rs#L433-L443

Reply:
Fixed!  Technically this code is unreachable because lscolors doesn't care about the file name of symlinks.  Still ought to be correct though.